.github/workflows/release.yml

```
@@ -89,6 +95,18 @@ jobs:
           asset_name: ${{ matrix.artifact-name }}.zip
           asset_content_type: application/zip
 
+      # TODO: Remove this after deprecation notice
+      - name: Upload Binary to Release aliases
+        if: ${{ matrix.artifact-alias == "" }}
```

I think this condition should be negated?
```suggestion
        if: ${{ matrix.artifact-alias != "" }}
```